docs/release-notes/winter-2019/README.md

```diff
@@ -45,20 +49,53 @@ As part of the 3.x.x Cedar release we moved the Icon SVG assets into their own r
 
 To resolve this issue we have chosen to leave the inline single icon components as part of @rei/cedar. These components have been updated to use the same SVG assets exported by [@rei/cedar-icons](https://github.com/rei/cedar-icons).
 
+### Modal Component
+
+TODO: ???
+
 ### Radio and Checkbox sizes
 
 Radio and checkbox now have a `size` prop with small, medium (default), and large variants. As part of this feature, the modifier `compact` has been deprecated and will be removed in the future in favor of using the new size prop.
 
 ## Bug Fixes
 
+### CdrButton
+
+- Icon fill is now updated for interactive and disabled states.
+
 ### CdrBreadcrumb
 
 - Applies focus to the first breadcrumb item when a truncated breadcrumb is expanded in order to support A11Y
 - Re-evaluates truncation logic when breadcrumb items are updated in order to support asynchronous loading
 
+### CdrDataTable
+
+- Updated styling logic so first row is always dark
+
+### CdrGrid, CdrRow, CdrCol
+
+- CdrRow and CdrCol have been updated so that clicks within the grid margin will no longer be blocked
+
+### CdrInput
+
+- Fixed padding of helper text
+
+
+### CdrPagination
```

Comment:
We have another section for this below that has notes -- is this one needed or is it just a dupe?

Reply:
wasn't sure what the bug was there exactly but it seems like a dupe